src/js/app/index.js

```diff
@@ -86,6 +87,22 @@ var vcsModeRefresh = function(eventData) {
   $('body').toggleClass('hgMode', !isGit);
 };
 
+var insertAlternateLinks = function(pageId) {
+  // For now pageId is null, which would link to the main page.
+  // In future if pageId is provided this method should link to a specific page
+
+  // The value of the hreflang attribute identifies the language (in ISO 639-1 format)
+  // and optionally a region (in ISO 3166-1 Alpha 2 format) of an alternate URL
+  var altLinks = _.map(LocaleStore.getSupportedLocales(), function(langCode) {
+    var url = "https://learngitbranching.js.org/?locale=" + langCode;
+    return '<link rel="alternate" hreflang="'+langCode+'" href="' + url +'" />';
```

Comment:
a lot of this will break if the langCode has a `"` for some reason. maybe we can uri encode or use jquery to set the attributes? will be cleaner than doing string concats

Reply:
Well, this code should work only with supported locales, so no way we are going to have `"` in the locales. I can escape the langCodes of course, but it looks like it is a bit of an overkill.